app/models/form_submission.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class FormSubmission < ApplicationRecord
+  # TODO: we need some real values here
+  enum form_type: {
+    option1: 0,
+    option2: 1
+  }
+
+  has_kms_key
+  has_encrypted :form_data, key: :kms_key, **lockbox_options
```

Comment:
@SamStuckey If this belongs to an `InProgressForm`, do we need to duplicate the form data here? Will it ever differ from its `InProgressForm`'s form data?

Reply:
yes, we do. InProgressForm data is deleted after 60 days. if we dont save it we wont be able to remediate old claims that have issues.

Reply:
yes holding onto old PII is bad, but as 526 remediation has proven, a necessary evil.  Until we are given an API endpoint that exposes our PII source of truth (some server in the depths of VA?) OR we are no longer culpable for failures, then we need to track this : (

As far as whey `InProgressForm` gets deleted, no idea